src/pages/ScoreReport.vue

```diff
@@ -128,8 +134,8 @@
             :lazy-pre-sorting="orderBy"
             data-cy="roar-data-table"
             @reset-filters="resetFilters"
-            @export-all="exportAll"
-            @export-selected="exportSelected"
+            @export-all="exportData({ includeProgress: false })"
```

Comment:
`includeProgress` defaults to `false` so we can drop this here? 

```suggestion
            @export-all="exportData"
```

Reply:
Yes, that was from the previous implementation, I forgot to delete.